openquake/hazardlib/gsim/bradley_2013.py

```diff
@@ -327,3 +328,90 @@ def _get_tvz_path_distance(self, rrup):
         """
 
         return rrup
+
+
+class Bradley2013LHC(Bradley2013):
+    """
+    Extend :class:`Bradley2013` to provide the model in terms of the larger
+    of two as-recorded horizontal components. This definition is required
+    by New Zealand building design standards.
+    """
+    DEFINED_FOR_INTENSITY_MEASURE_COMPONENT = \
+        const.IMC.GREATER_OF_TWO_HORIZONTAL
+
+    #: This implementation is non-verified because this version of the
+    #: model has not been published, nor is independent code available.
+    non_verified = True
+
+    def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
+        """
+        See :meth:`superclass method
+        <.base.GroundShakingIntensityModel.get_mean_and_stddevs>`
+        for spec of input and result values.
+        """
+        # extracting dictionary of coefficients specific to required
+        # intensity measure type.
+        C = self.COEFFS[imt]
+        # intensity on a reference soil is used for both mean
+        # and stddev calculations.
+        ln_y_ref = self._get_ln_y_ref(rup, dists, C)
+        # exp1 and exp2 are parts of eq. 7
+        exp1 = np.exp(C['phi3'] * (sites.vs30.clip(-np.inf, 1130) - 360))
+        exp2 = np.exp(C['phi3'] * (1130 - 360))
+        # v1 is the period dependent site term. The Vs30 above which, the
+        # amplification is constant
+        v1 = self._get_v1(imt)
+
+        mean = self._get_mean(sites, C, ln_y_ref, exp1, exp2, v1)
+        mean += self._convert_to_LHC(imt)
+
+        stddevs = self._get_stddevs(sites, rup, C, stddev_types,
+                                    ln_y_ref, exp1, exp2)
+        return mean, stddevs
+
+    def _convert_to_LHC(self, imt):
```

Comment:
Hi Marco
Long term, it might be beneficial for the user to define the intensity measure component in the .ini file, but in the interim, would you prefer that the independent function is in this gsim file, or elsewhere?
Thanks
Chris

Reply:
Hi Chris, I am fine with a temporary solution like the one you propose. I will work to something more flexible in the future. I'll ask you help!! ... btw we will need to add tests.
Cheers,
Marco

Reply:
Happy to help when the time comes. 
For the tests, are you referring to unit tests around the independent function 'convert_to_LHC', or a set of results on which to test the application of the function to a series of inputs?